dev/reference/tools/error-case-handling/load-misuse-handling.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Common Misuses During Full Data Import
+category: reference
+---
+
+# Common Misuses During Full Data Import
+
+This document introduces the common error scenarios in using [Loader](/dev/reference/tools/loader.md) or [TiDB Data Migration](/dev/reference/tools/data-migration/overview.md) (DM) during the full data import process, and also provides their reasons and solutions.
+
+## Error: ```Try adjusting the `max_allowed_packet` variable```
+
+The following error is reported during full data import:
+
+```
+packet for query is too large. Try adjusting the 'max_allowed_packet' variable
+```
+
+### Reasons
+
+* Both MySQL client and MySQL/TiDB Server have `max_allowed_packet` quotas. If any of the `max_allowed_packet` quotas is violated, the client receives a corresponding error message. Currently, the latest version of Syncer, Loader, DM, and TiDB Server all have a default `max_allowed_packet` quota of `64M`.
+    * Please use the latest version, or the latest stable version of the tool. See the [download page](/dev/reference/tools/download.md).
+* The full data import processing module in Loader or DM Loader does not support splitting `dump sqls` files. It is because Mydumper uses the simplest code to implement, as stated in the comment `/* Poor man's data dump code */`. If you split files in Loader or DM, you need to implement a complete parser based on `TiDB parser` to handle data splits correctly, which causes the following problems:
```

Comment:
```suggestion
* The full data import processing module in Loader or DM does not support splitting `dump sqls` files. It is because Mydumper uses the simplest code to implement, as stated in the comment `/* Poor man's data dump code */`. If you split files in Loader or DM, you need to implement a complete parser based on `TiDB parser` to handle data splits correctly, which causes the following problems:
```